spark/src/main/scala/io/delta/sharing/spark/DeltaSharingClient.scala

```diff
@@ -255,8 +266,12 @@ class DeltaSharingRestClient(
 
         val statusCode = status.getStatusCode
         if (statusCode != HttpStatus.SC_OK) {
+          var additionalErrorInfo = ""
+          if (statusCode == HttpStatus.SC_FORBIDDEN && tokenExpired(profile)) {
+            additionalErrorInfo = s"It may be caused by expired token:${profile.expirationTime}"
```

Comment:
`It may be caused by an expired token as it has expired at ${profile.expirationTime}`.

Reply:
updated.